src/main/java/com/CezaryZal/api/body/manager/BodySizeService.java

```diff
@@ -0,0 +1,58 @@
+package com.CezaryZal.api.body.manager;
+
+import com.CezaryZal.api.body.BodySizeRepository;
+import com.CezaryZal.api.body.entity.BodySize;
+import com.CezaryZal.api.body.entity.BodySizeDao;
+import com.CezaryZal.api.body.entity.BodySizeDto;
+import com.CezaryZal.api.body.manager.mapper.ConverterBodySizeToDto;
+import com.CezaryZal.api.body.manager.mapper.ConverterDaoToBodySize;
+import com.CezaryZal.api.body.manager.repo.BodySizeRepoService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class BodySizeService extends BodySizeRepoService {
```

Comment:
@Service rozrszerza @Service?

Reply:
Tak, *RepoService jest bezpośrednio związane z interfacem repository i go obsługuje.
A *Service może wykonać wszystkie metody nadklasy oraz metody convertujące obiekty itp.

Reply:
Ok, dopiero zczaiłem.
Myślałem że jeśli wstrzykuje się zależność to jest wymagana adnotacja